src/transaction-runner.coffee

```diff
@@ -352,7 +352,11 @@ class TransactionRunner
     # See also https://github.com/joyent/node/issues/2216
     segments = [serverPath, requestPath]
     segments = (segment.replace(/^\/|\/$/g, '') for segment in segments)
-    return '/' + segments.join('/')
+    # Keep trailing slash at the end if specified in requestPath
+    # and if requestPath isn't only '/'
+    # https://github.com/apiaryio/dredd/issues/93
+    trailingSlash = (requestPath != '/' and requestPath.slice(-1) == '/' and '/' or '')
```

Comment:
For the `and '/' or ''`, that was my attempt at doing a one-liner for:
```coffeescript
if requestPath isnt '/' and requestPath.splice(-1) is '/'
	trailingSlash = '/'
else
	trailingSlash = ''
```

After digging a bit I found out that it's usually done with:
```coffeescript
trailingSlash = if requestPath isnt '/' and requestPath.slice(-1) is '/' then '/' else ''
```

Thanks for your comment.